test/lib/ansible_test/_data/pytest/plugins/ansible_pytest_unit_test_module_mocks.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017–2021 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Pytest fixtures for mocking Ansible modules."""
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from io import BytesIO
+import json
+import sys
+
+import pytest
+
+import ansible.module_utils.basic
+from ansible.module_utils.common._collections_compat import MutableMapping
+from ansible.module_utils.six import PY3, string_types
+from ansible.module_utils._text import to_bytes
+
+
+@pytest.fixture
+def ansible_module_args(mocker, monkeypatch, request):
+    """Patch and return stdin buffer with module args."""
+    monkeypatch.setattr(ansible.module_utils.basic, '_ANSIBLE_ARGS', None)
```

Comment:
_Self note:_ Why don't we just patch `_ANSIBLE_ARGS` or maybe `AnsibleModule._load_params()`?

Reply:
Okay... patching `AnsibleModule._load_params()` and `AnsibleModule.params` would bypass input pre-validation with early exits but patching `_ANSIBLE_ARGS` would probably be fine.